lockbox-ios/Store/AutoLockStore.swift

```diff
@@ -63,11 +63,7 @@ class AutoLockStore {
         if evt.element == .background {
```

Comment:
I would remove this conditional entirely rather than keeping the pointless subscription.

Reply:
@sashei Ah ok, so if I understand you correctly: remove all of (currently) lines 63-72 and leave it empty like this?

```private func lifecycleAction(evt: Event<LifecycleAction>) {}```